docs/2022/registration.md

```diff
@@ -0,0 +1,16 @@
+# OSG All-Hands Meeting 2020 &ndash; Registration
+
+## Costs
+
+Registration for the OSG All-Hands Meeting 2022 costs **$70**, which includes:
+
+* Attendance Monday through Wednesday, (Thursday and Friday are Virtual only)
+* morning snack/light breakfast, afteroon snacks, and beverages throughout
+
+
+## Registration
+
+The registration system will be linked from here very soon.
+
+When registering, please indicate which days you plan to attend so that we have
+accurate counts for catering and so forth.
```

Comment:
It would be good to include the sentence or two about refunds if OSG cancels here.